frontend/src/libs/components/input/input.tsx

```diff
@@ -23,17 +28,65 @@ const Input = <T extends FieldValues>({
   name,
   placeholder = '',
   type = 'text',
+  isActive,
+  isFilled,
+  isDisabled,
 }: Properties<T>): JSX.Element => {
   const { field } = useFormController({ name, control });
 
   const error = errors[name]?.message;
   const hasError = Boolean(error);
+  const mapStatuses = (): string => {
+    let classNames = '';
+
+    if (isActive) {
+      classNames += ' ' + styles.active;
+    }
+
+    if (isFilled) {
+      classNames += ' ' + styles.filled;
+    }
+
+    if (isDisabled) {
+      classNames += ' ' + styles.disabled;
+    }
+
+    if (error) {
+      classNames += ' ' + styles.error;
+    }
+
+    return classNames;
+  };
 
   return (
-    <label>
+    <label className={styles.label}>
       <span>{label}</span>
-      <input {...field} type={type} placeholder={placeholder} />
-      {hasError && <span>{error as string}</span>}
+      {type !== 'textarea' && (
+        <input
+          {...field}
+          type={type}
+          placeholder={placeholder}
```

Comment:
create constant inputArea before rendering jsx, and depending which type you get, return needed jsx